dashboard/app/templates.html

```diff
@@ -86,14 +86,15 @@ <h1><a href="/{{$.Namespace}}">syzbot</a></h1>
 <table class="list_table">
 	{{if $.Caption}}
 		{{if $.Fragment}}
-			<caption id="{{$.Fragment}}"><a class="plain" href="#{{$.Fragment}}">
+			<caption id="{{$.Fragment}}"><a class="plain" href="?fragment={{$.Fragment}}">
```

Comment:
Is the intention to use it while working with the site, or to give links to other people?

Reply:
In the email it was requested just that the feature be present. Ideally I think it should be suitable for both purposes you described